src/malcolmWidgets/attributeDetails/attributeSelector/attributeSelector.component.js

```diff
@@ -24,21 +36,43 @@ const AttributeSelector = props => {
         return <TextUpdate Text={props.attribute.value} />;
       }
     }
-  }
 
+    if (tags.some(t => t === 'widget:checkbox')) {
```

Comment:
I think you want to merge in the current version for this as I've changed it so there is a widgetTag you can use, it means we can start moving towards a switch statement rather than if/else if/else